src/Storages/Kafka/KafkaConfigLoader.cpp

```diff
@@ -374,6 +378,11 @@ void updateGlobalConfiguration(
                 kafka_object_config.get(client_id_key),
                 facility,
                 message);
+            if (client_logs_level <= DB::LogsLevel::error)
+            {
+                if (auto sink_shared_ptr = sink.lock())
+                    sink_shared_ptr->setExceptionInfo(message, /* with_stacktrace = */ true);
```

Comment:
Maybe we should also increment KafkaConsumerErrors?

Reply:
I am not sure about semantic of ProfileEvents::KafkaConsumerErrors. 
It looks like a counter of erroneous messages that were consumed.
Although the Description is "Number of errors reported by librdkafka during polls"

Shouldn't we make the description more generic?

We can handle this counter inside setExceptionInfo, but it is not set e.g. for Producers.

Reply:
> I am not sure about semantic of ProfileEvents::KafkaConsumerErrors.
It looks like a counter of erroneous messages that were consumed.
Although the Description is "Number of errors reported by librdkafka during polls"
>
> Shouldn't we make the description more generic?

I agree, that counter is for errorneous messages. And yes, the description is too generic.

> We can handle this counter inside setExceptionInfo, but it is not set e.g. for Producers.

I think we should introduce a new counter for this. I am not sure about the naming, because `KafkaConsumerErrors` should be `KafkaErrorneousMessages`, but I am not sure if introducing a (slight, but still) breaking change is reasonable or not.

Reply:
Well actually KafkaConsumerErrors is currently incremented if INSTEAD of the message we get something different from librdkafka (it can mix in own errors in the stream of messages). 

Broken / unparsable messages counted by KafkaMessagesFailed 

So i opt for reusing the KafkaConsumerErrors here.